File: CampusServer/src/main/java/com/cai/campus/server/entity/UserSignInDetail.java
package com.cai.campus.server.entity;

public class UserSignInDetail {
    private SignIn signIn;
    private SignInRecord record;
    private String groupName;

    public SignIn getSignIn() {
        return signIn;
    }

    public void setSignIn(SignIn signIn) {
        this.signIn = signIn;
    }

    public SignInRecord getRecord() {
        return record;
    }

    public void setRecord(SignInRecord record) {
        this.record = record;
    }

    public String getGroupName() {
        return groupName;
    }

    public void setGroupName(String groupName) {
        this.groupName = groupName;
    }
}

File: CampusClient/app/src/main/java/com/cai/campus/common/network/model/UserSignInDetail.kt
package com.cai.campus.common.network.model

import java.io.Serializable

class UserSignInDetail : Serializable {
    var signIn: SignIn? = null
    var record: SignInRecord? = null
    var groupName: String? = null

    companion object {
        private const val serialVersionUID = 628352243016151750L
    }

    override fun toString(): String {
        return "UserSignInDetail(signIn=$signIn, record=$record, groupName=$groupName)"
    }


}